spec/miscellaneous/lib/works_owner_spec.rb

```diff
@@ -81,6 +81,8 @@
       end
 
       it "should change after a work is orphaned" do
+        # Ensure that the orphan account exists:
+        build(:user, login: "orphan_account").save
```

Comment:
Unfortunately, no. It does try to create it:
https://github.com/otwcode/otwarchive/blob/854a4926c67c0419b10b1a4fe2165d8ccb8e9d87/app/models/user.rb#L547-L553
But because the record that it builds is missing a lot of basic info (like email and password), the new user doesn't get saved.

Reply:
I see how it works in feature tests now:

https://github.com/otwcode/otwarchive/blob/854a4926c67c0419b10b1a4fe2165d8ccb8e9d87/features/step_definitions/user_steps.rb#L11-L14

Not ideal, but we don't have to deal with this now.

Reply:
I made [AO3-5725](https://otwarchive.atlassian.net/browse/AO3-5725).